dotnet/samples/Demos/ProcessFrameworkWithAspire/ProcessFramework.Aspire/ProcessFramework.Aspire.ProcessOrchestrator/TranslatorAgentHttpClient.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Text;
+using System.Text.Json;
+using ProcessFramework.Aspire.Shared;
+
+namespace ProcessFramework.Aspire.ProcessOrchestrator;
+
+public class TranslatorAgentHttpClient(HttpClient httpClient)
+{
+    private static readonly Uri s_agentUri = new("/api/translatoragent");
+
+    public async Task<string> TranslateAsync(string textToTranslate)
+    {
+        var payload = new TranslationRequest { TextToTranslate = textToTranslate };
+        var response = await httpClient.PostAsync(s_agentUri, new StringContent(JsonSerializer.Serialize(payload), Encoding.UTF8, "application/json"));
+        response.EnsureSuccessStatusCode();
+        var responseContent = await response.Content.ReadAsStringAsync();
```

Comment:
nit: we usually add `.ConfigureAwait(false)` in all async calls, to be on the safe side